### PR TITLE
PP 618 fix cert loading

### PR DESCRIPTION
**Pull request by ijmad**
There is no need for the private keys.
cacerts is a truststore, not a keystore.

Move to a programmatic approach for creating a truststore from the pem directory, to avoid the need for bash scripting inside the containers.

### CODE REVIEW

File: swagger/swagger.json [hunk 1]
```diff
@@ -302,6 +297,11 @@
           "type" : "string",
           "example" : "worldpay",
           "readOnly" : true
+        },
+        "status" : {
+          "type" : "string",
+          "example" : "CREATED",
+          "readOnly" : true
```
Comment (ijmad) [inline swagger/swagger.json]: I'm not really sure why this change has moved this attribute in swagger, but it seems harmless enough.

File: swagger/swagger.json [hunk 2]
```diff
@@ -267,29 +267,29 @@
           "example" : "your-reference",
           "readOnly" : true
         },
-        "created_date" : {
+        "description" : {
```
Comment (kakumara) [inline swagger/swagger.json]: I cannot see you've done any changes to Documentation/Examples. Are these intentional changes?

File: src/main/java/uk/gov/pay/api/validation/URLValidator.java [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 public enum URLValidator {
     SECURITY_ENABLED {
-        private final UrlValidator URL_VALIDATOR = new UrlValidator(new String[]{"https"});
+        private final UrlValidator URL_VALIDATOR = new UrlValidator(new String[]{"https"}, ALLOW_LOCAL_URLS);
```
Comment (kakumara) [inline src/main/java/uk/gov/pay/api/validation/URLValidator.java]: Is this a good thing? This means that If someone sends us a `localhost`  `return_url` we accept it?

Comment (ijmad) [inline src/main/java/uk/gov/pay/api/validation/URLValidator.java]: Yeah, it's a bit controversial, but people will be coming from localhost for sandbox testing. Maybe we need some more discussion. I'll withdraw this change for now.

Comment (kakumara) [inline src/main/java/uk/gov/pay/api/validation/URLValidator.java]: 👍
